src/gui/dialogs/addon/manager.cpp

```diff
@@ -391,6 +396,50 @@ void addon_manager::pre_show(window& window)
 	connect_signal_notify_modified(type_filter,
 		std::bind(&addon_manager::apply_filters, this));
 
+	// Language filter
+	// Clear current language filter vector, lest all entries are shown twice
+	addon_manager::language_filter_types_.clear();
```

Comment:
If this is a regular class member, no need to clear it in pre-show.

Reply:
:ok_hand: